.github/workflows/e2e-deleteme.yaml

```diff
@@ -0,0 +1,49 @@
+name: E2E Tests
+
+on:
+  pull_request:
+
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+  run-e2e-tests:
+    name: Run E2E Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ginkgo_focus:
+          #- "KCP remediation"
+          #- "MachineDeployment remediation"
+          - "Workload cluster creation"
+          #- "Workload cluster scaling"
+          #- "Workload cluster upgrade"
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          audience: sts.amazonaws.com
+          aws-region: us-east-2
+          role-to-assume: arn:aws:iam::018302341396:role/GithubOIDC
+          role-duration-seconds: 3600
+      - name: Check out repo
+        uses: actions/checkout@v4
+      - name: Setup tmate session
+        uses: mxschmitt/action-tmate@v3
+        with:
+          detached: true
+      - name: Install requirements
+        run: |
+          sudo apt install make
+          sudo apt install wget
+      - name: Increase inotify watches
+        run: |
+          # Prevents https://cluster-api.sigs.k8s.io/user/troubleshooting#cluster-api-with-docker----too-many-open-files
+          sudo sysctl fs.inotify.max_user_watches=1048576
+          sudo sysctl fs.inotify.max_user_instances=8192
+      - name: Run e2e tests
+        run: |
+          sudo -E ./hack/ci-e2e-tests.sh true aws v0.1.2
```

Comment:
I can't understand this `true` here, would elaborate a bit? I see `${0}`, `${2}` and `${3}` in the script, this should be `${1}`, right?